src/spacesettlers/clients/RandomTeamClient.java

```diff
@@ -47,70 +52,68 @@ public void shutDown(Toroidal2DPhysics space) {
 
 	}
 
-
 	@Override
 	public Map<UUID, AbstractAction> getMovementStart(Toroidal2DPhysics space,
 			Set<AbstractActionableObject> actionableObjects) {
 		HashMap<UUID, AbstractAction> randomActions = new HashMap<UUID, AbstractAction>();
-		
-		
-		for (AbstractObject actionable :  actionableObjects) {
+
+		for (AbstractObject actionable : actionableObjects) {
 			if (actionable instanceof Ship) {
 				Ship ship = (Ship) actionable;
 				AbstractAction current = ship.getCurrentAction();
-				
+
 				// if we finished, make a new spot in space to aim for
 				if (current == null || current.isMovementFinished(space)) {
 					Position currentPosition = ship.getPosition();
-					Position newGoal = space.getRandomFreeLocationInRegion(random, Ship.SHIP_RADIUS, (int) currentPosition.getX(), 
-							(int) currentPosition.getY(), RANDOM_MOVE_RADIUS);
-					MoveAction newAction = null;
-					newAction = new MoveAction(space, currentPosition, newGoal);
-					//System.out.println("Ship is at " + currentPosition + " and goal is " + newGoal);
-					SpacewarGraphics graphic = new CircleGraphics(1, getTeamColor(), newGoal);
-					graphics.add(graphic);
-					//Vector2D shortVec = space.findShortestDistanceVector(currentPosition, newGoal);
-					//LineShadow lineShadow = new LineShadow(currentPosition, newGoal, shortVec);
-					//newShadows.add(lineShadow);
+					Position newGoal = space.getRandomFreeLocationInRegion(random, Ship.SHIP_RADIUS,
+							(int) currentPosition.getX(), (int) currentPosition.getY(), RANDOM_MOVE_RADIUS);
+					// MoveAction newAction = null;
+					AbstractAction newAction = null;
+					// newAction = new MoveAction(space, currentPosition, newGoal);
+					newAction = new RawAction(new Vector2D(600, 0), 0);
```

Comment:
Why is the random ship using raw action?  I don't want to leave it that way.  MoveAction is a higher level action and was intended to show the students how to use MoveAction.